dev/Scroller/Scroller.cpp

```diff
@@ -5545,6 +5561,302 @@ void Scroller::ChangeZoomFactorWithAdditionalVelocityPrivate(
     }
 }
 
+void Scroller::ProcessPointerWheelScroll(
```

Comment:
nit: is it possible to move the pre-rs5 logic into a separate file ? I wonder if that would make this a bit simpler ?

Reply:
There's a bunch of RS2/RS4/RS5 specific code in Scroller.cs - I'll just leave it there for now.